examples/ppo_translation_t5.py

```diff
@@ -0,0 +1,197 @@
+"""Example of using PPO to train a T5 model for translation.
+Based on examples/summarize_daily_cnn/t5_summarize_daily_cnn.py"""
+
+import os
+from typing import List
+
+import torch
+from datasets import load_dataset
+from tqdm import tqdm
+from transformers import AutoTokenizer
+
+import trlx
+from trlx.data.configs import (
+    ModelConfig,
+    OptimizerConfig,
+    SchedulerConfig,
+    TokenizerConfig,
+    TrainConfig,
+    TRLConfig,
+)
+from trlx.models.modeling_ppo import PPOConfig
+
+try:
+    import evaluate
+except ImportError:
+    raise ImportError(
+        "To run this example, please install the `evaluate` and `nltk` packages" "by running `pip install evaluate`"
+    )
+
+
+config = TRLConfig(
+    train=TrainConfig(
+        seq_length=612,
+        epochs=100,
+        total_steps=100000,
+        batch_size=12,
+        checkpoint_interval=10000,
+        eval_interval=500,
+        pipeline="PromptPipeline",
+        trainer="AcceleratePPOTrainer",
+        tracker="wandb",
+    ),
+    model=ModelConfig(
+        model_path="t5-large",
+        model_arch_type="seq2seq",
+        num_layers_unfrozen=2,
+    ),
+    tokenizer=TokenizerConfig(
+        tokenizer_path="t5-large",
+        truncation_side="right",
+    ),
+    optimizer=OptimizerConfig(
+        name="adamw",
+        kwargs={
+            "lr": 1.0e-5,
+            "betas": [0.9, 0.999],
+            "eps": 1.0e-8,
+            "weight_decay": 1.0e-6,
+        },
+    ),
+    scheduler=SchedulerConfig(
+        name="cosine_annealing",
+        kwargs={
+            "T_max": 10000,
+            "eta_min": 1.0e-6,
+        },
+    ),
+    method=PPOConfig(
+        name="PPOConfig",
+        num_rollouts=512,
+        chunk_size=12,
+        ppo_epochs=4,
+        init_kl_coef=0.05,
+        target=6,
+        horizon=10000,
+        gamma=0.99,
+        lam=0.95,
+        cliprange=0.2,
+        cliprange_value=0.2,
+        vf_coef=1.0,
+        scale_reward=None,
+        ref_mean=None,
+        ref_std=None,
+        cliprange_reward=10,
+        gen_kwargs={
+            "max_new_tokens": 100,
+        },
+        gen_experience_kwargs={
+            "max_new_tokens": 100,
+            "do_sample": False,
+            "num_beams": 4,
+            "temperature": 1.0,
+        },
+    ),
+)
+
+# COMET is the metric we are optimizng for
+comet_metric = evaluate.load("comet", "wmt20-comet-da", progress_bar=False)
```

Comment:
I'm having trouble loading comet and not sure if this is a general issue. Is `wmt20-comet-da` stored in a private repo? See repro here: https://colab.research.google.com/drive/1Du3-sIff2mz-34I62NP2pjmRRezTPGtH?usp=sharing

Reply:
Huum, I was able to reproduce your issue as well. It seems the issue with with the newest `comet` package. If you install it with `unbabel-comet==1.1.3`, my code works. With the newest version of `unbabel-comet==2.0.0` which is the one you installed, they removed the `wmt20-comet-da` model and replaced it with a newer performing model called `Unbabel/wmt22-comet-da`.

Since I've only ran with the `wmt20-comet-da`, I think it would be best to stick with the `unbabel-comet==1.1.3` and use `wmt20-comet-da`. Should I add a version check for `unbabel-comet` in the `ppo_translation_t5.py` file?

Reply:
Thanks and done!